pkg/kubelet/network/exec/exec.go

```diff
@@ -33,7 +33,7 @@ limitations under the License.
 //                formatted output indicating the pod's IPAddress(v4/v6). An empty string value or an erroneous output
 //                will mean the container runtime (docker) will be asked for the PodIP
 //                e.g. {
-//                         "apiVersion" : "v1beta1",
+//                         "apiVersion" : "v1",
```

Comment:
This struct represents version "v1" which is noted at https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/network/plugins.go#L60，What's this note mean or just a wrong note?